client/basic-authorship/src/basic_authorship.rs

```diff
@@ -473,7 +474,8 @@ where
 							 so we will try a bit more before quitting."
 						);
 					} else {
-						debug!("Block is full, proceed with proposing.");
+						debug!("Block is full, proceeding with proposing.");
```

Comment:
Shouldn't we specify what full mean here? As there are 2 limits, weight and size.